test/unit/resources/aws_s3_bucket_object_test.rb

```diff
@@ -99,9 +99,17 @@ def test_property_object_acl_public
     end
     refute_empty(public_grants)
   end
+
+  def test_resource_id
+    assert_equal(@bucket_object.resource_id, 'public_bucket_public.file')
+  end
+
+  def test_resource_id_not_nil_check
```

Comment:
This is redundant negative assertion since the above assertion always asserts against a valid value